gpg-mailgate.py

```diff
@@ -153,7 +154,12 @@ def gpg_decrypt_all_payloads( message ):
 		# Check if message is PGP/INLINE encrypted without attachments (or unencrypted without attachments)
 		else:
 			decrypted_message = decrypt_inline_without_attachments(decrypted_message)
-
+	if cfg['gpg']['attach_original'] == "yes" and success:
+		attachment = MIMEBase('text', 'plain')
+		attachment.set_payload(message.as_string())
+		encoders.encode_base64(attachment)
+		attachment.add_header('Content-Disposition', 'attachment; filename="original-mail.txt"')
```

Comment:
Nice enhancement! To prevent collisions of attachment names, can we check first if a file with name `original-mail.txt` already exists?

Reply:
And append a 1,2,3,... to the filename on collision or a randomseed on collision?

Reply:
Valid point. I'll update the pull request.